### PR TITLE
Re-organizing tests to be beside the samples they test.

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

File: appengine/ndb/overview/main_test.py [hunk 1]
```diff
@@ -12,14 +12,13 @@
...
-# from the app main.py
-from appengine.memcache.guestbook import main
 from tests import AppEngineTestbedCase
 import webtest
 
+from . import main
 
-class TestHandlers(AppEngineTestbedCase):
 
+class TestHandlers(AppEngineTestbedCase):
```
Comment (tmatsuo) [inline appengine/ndb/overview/main_test.py]: I don't know why this line is shown in the diff.

Comment (jonparrott) [inline appengine/ndb/overview/main_test.py]: *shrugs*, me either. I didn't change it.

**Comment by tmatsuo** [comment]
LGTM, merging

**Comment by jonparrott** [comment]
Thanks!

**Pull request merged**